README.md

```diff
@@ -218,15 +236,28 @@ grunt custom --name="myPackage"
 
 ### Run Tests On Your Computer
 
-To run both the browser and node tests for this package, use the command `npm test` or `yarn test`.
+To run both the browser and node tests for this package:
 
-To run only the node tests, use the command `npm run test:node` or `yarn run test:node`.
+```bash
+npm test
+```
 
-To run only the browser tests, use the command `npm run test:browser` or `yarn run test:browser`.  Any browsers that
-Testem finds on your system will be launched sequentially with each browser running the full Infusion test suite. The
-results will be returned in your terminal in the [TAP](https://testanything.org/) format. Once you have run
-`npm install`, you can use the command ``node node_modules/testem/testem.js launchers`` from the root of this repository
-to get a list of browsers that Testem can launch on your system.
+To run only the node tests:
+
+```bash
+npm run test:node
+```
+
+To run only the browser tests:
+
+```bash
+npm run test:browser
+```
+
+Any browsers that Testem finds on your system will be launched sequentially with each browser running the full Infusion
+test suite. The results will be returned in your terminal in the [TAP](https://testanything.org/) format. Once you have
+run `npm install`, you can use the command `npx testem launchers` from the root of this repository to get a list of
+browsers that Testem can launch on your system.
```

Comment:
This is no longer completely accurate. We actually only attempt to run the browser tests in Headless Chrome and Headless Firefox. We may modify that later, but that's it for now.

https://github.com/fluid-project/infusion/blob/8dfc99ed2c0c8f7e28b90419862f0d7f752c7cc8/tests/testem.js#L28-L29

Reply:
@jobara should the instructions about using `npx testem launchers` be removed as it's no longer particularly helpful?

Reply:
Hmm.. probably. It could help if an expected browser isn't running or if you wanted to see what else was available, but that's probably it.